runtime-modules/forum/src/lib.rs

```diff
@@ -200,6 +200,31 @@ pub struct PollAlternative<Hash> {
     pub vote_count: u32,
 }
 
+/// Represents all poll alternative texts and vote count for each one
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug)]
+pub struct PollAlternativeInput {
+    /// alternative description
+    pub alternative_text: Vec<u8>,
+
+    /// Vote count for the alternative
+    pub vote_count: u32,
+}
+
+/// Represents a poll input
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug)]
+pub struct PollInput<Timestamp> {
+    /// description
+    pub description: Vec<u8>,
+
+    /// pallet_timestamp of poll end
```

Comment:
Fix this comment, please.

Reply:
Done

Reply:
https://github.com/Joystream/joystream/pull/2531/commits/4bb08fdb7c9a5c49976eb313b279b62aa8973780